src/vstart.sh

```diff
@@ -1072,67 +1072,27 @@ EOF
 # https://launchpad.net/~nfs-ganesha/+archive/ubuntu/nfs-ganesha-2.7
 
 start_ganesha() {
-    GANESHA_PORT=$(($CEPH_PORT + 4000))
+    #GANESHA_PORT=$(($CEPH_PORT + 4000))
     local ganesha=0
 
     for name in a b c d e f g h i j k l m n o p
     do
         [ $ganesha -eq $GANESHA_DAEMON_NUM ] && break
 
-        port=$(($GANESHA_PORT + ganesha))
+        #port=$(($GANESHA_PORT + ganesha))
+        port=2049
```

Comment:
You probably don't want to use 2049 here. I'd just let it pick a random ephemeral port for vstart as that won't conflict with a "real" nfs server that might be running on the same box.

Reply:
Can the random port be set for vstart without moving the entire NFS_CORE_PARAM block into bootstrap config from common config? 
```
-       echo "%url rados://nfs-ganesha/tester/a" > "$ganesha_dir/ganesha.conf"
+       echo "NFS_CORE_PARAM {
+       Enable_NLM = false;
+       Enable_RQUOTA = false;
+       Protocols = 4;
+       NFS_Port = $port;
+}
+
+%url rados://nfs-ganesha/tester/a" > "$ganesha_dir/ganesha.conf"
```
I tried setting only the port in bootstrap config and rest in common config. I found the port to be set to 2049 with  NFS_CORE_PARAM block being read twice.

Reply:
Placing NFS_CORE_PARAM block after rados url fixes this issue. Thanks for the hack Jeff.